README.md

```diff
@@ -126,9 +126,11 @@ of extensions are being processed such that the *base-instruction* is not includ
 
 The following artifacts can be generated using parse.py:
 
-- instr\_dict.yaml : This is file generated always by parse.py and contains the
-  entire main dictionary `instr\_dict` in YAML format. Note, in this yaml the
-  *dots* in an instruction are replaced with *underscores*
+- instr\_dict.json : This is file generated always by parse.py and contains the
```

Comment:
@Timmmm I think the above can be replaced by _(just some grammatical improvements)_

_"This file is always generated by `parse.py` and contains the.... "_

Reply:
Ooo good spot; I'll fix this and the other one in #304.

Reply:
@aswaterman I think @Timmmm has pushed the above changes in #304